hazelcast-client-legacy/src/main/java/com/hazelcast/client/spi/impl/listener/ClientEventRegistration.java

```diff
@@ -29,6 +31,7 @@
 
     public ClientEventRegistration(String serverRegistrationId,
                                    int callId, Address subscriber) {
+        isNotNull(serverRegistrationId, "serverRegistrationId");
```

Comment:
It can be simplified:

this.serverRegistrationId = checkNotNull(serverRegistrationId, "serverRegistrationId");

But doesn't need to be changed now.